### PR TITLE
Add missing senators from 116th Congress

### LINKED ISSUE #167: Add missing senators from the 116th congress
Here's a list of the newly-seated senators who are still missing photos, and a possible photo for each. I'm happy to submit a PR with these photos, if that would be helpful to get them merged in.

Thanks!

- [Rick Scott - S001217](https://www.budget.senate.gov/imo/media/image/Official-Portrait_RLS.jpg) (This photo is available on the [Bioguide](http://bioguide.congress.gov/scripts/biodisplay.pl?index=S001217) and [Budget committee site](https://www.budget.senate.gov/about/committee-members), and [Wikipedia lists it](https://en.wikipedia.org/wiki/Rick_Scott#/media/File:Sen._Rick_Scott_(R-FL)_official_Senate_portrait.jpg) as public domain)
Rick Scott

- [Mitt Romney - R000615](https://upload.wikimedia.org/wikipedia/commons/7/7f/Mitt_Romney_official_US_Senate_portrait.jpg) (See #166)
Mitt Romney

- [Joshua Hawley - H001089](https://www.judiciary.senate.gov/imo/media/image/Hawley_Josh-010919-RMB-042a04_300.png) (This photo is available on [Sen. Hawley's website](https://hawley.senate.gov) and the [Judiciary Committee site](https://www.judiciary.senate.gov), and [Wikipedia lists it](https://en.wikipedia.org/wiki/Josh_Hawley#/media/File:Josh_Hawley_official_photo.jpg) as public domain)
Josh Hawley

- [Mike Braun - B001310](https://upload.wikimedia.org/wikipedia/commons/thumb/2/25/Mike_Braun%2C_Official_Portrait%2C_116th_Congress.jpg/1280px-Mike_Braun%2C_Official_Portrait%2C_116th_Congress.jpg) (A [Wikipedia user](https://commons.wikimedia.org/wiki/File:Mike_Braun,_Official_Portrait,_116th_Congress.jpg) found this full-sized version of the photo on the [Health, Education, Labor, and Pensions committee site](https://www.help.senate.gov/about/members))
Mike Braun

### PR DESCRIPTION
**Pull request by pandringa**
This closes #167 by adding images for Rick Scott, Mitt Romney, Josh Hawley, and Mike Braun.

I used a reverse Google Image search to find full-res, uncropped versions of the Rick Scott and Mike Braun photos, and left a comment in their metadata files to note where the full-res image came from. If there's a better way to denote that, let me know and I'd be happy to fix.

* [x] *I'm acknowledging that my submission is free of all known copyright in at least the United States.* (In general, works of the federal government should satisfy this criteria.)

### CODE REVIEW

File: congress/metadata/B001310.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: Members | About | The U.S. Senate Committee on Health, Education, Labor & Pensions
```
Comment (hugovk) [inline congress/metadata/B001310.yaml]:
```suggestion
name: Senate Committee on HELP
```

File: congress/metadata/H001089.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+name: Members - Senate Committee on the Judicary
```
Comment (hugovk) [inline congress/metadata/H001089.yaml]:
```suggestion
name: Senate Committee on the Judiciary 
```

File: congress/metadata/R000615.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+name: About Mitt | Senator Mitt Romney
```
Comment (hugovk) [inline congress/metadata/R000615.yaml]:
```suggestion
name: Office of Mitt Romney
```

**Review by hugovk — COMMENTED** [review]
Thanks for the PR!

We should document the fields a bit better, we're looking for a short noun and a link to whatever the noun refers to (re: https://github.com/unitedstates/images/pull/39#issuecomment-102656761). Maybe something like this?

**Comment by pandringa** [comment]
Ah ok, that makes sense - I was a little unclear on what the meta fields were for. Thanks for those corrections!

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**